src/main/java/it/gov/pagopa/payhub/activities/dto/export/debtposition/InstallmentExportFlowFileDTO.java

```diff
@@ -0,0 +1,169 @@
+package it.gov.pagopa.payhub.activities.dto.export.debtposition;
+
+import com.opencsv.bean.CsvBindByName;
+import it.gov.pagopa.payhub.activities.enums.EntityIdentifierType;
+import it.gov.pagopa.payhub.activities.enums.UniqueIdentifierType;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.math.BigDecimal;
+import java.time.OffsetDateTime;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
+public class InstallmentExportFlowFileDTO {
+
+    @CsvBindByName(column = "IUF")
```

Comment:
Use `profiles` property to define the version of the track which support the field.
When configuring the CsvReader you will have to set the profile related to the track version.
Let's introduce a constant for each version in order to centralize the profile value (which will result to be aligned with track version)
EXPORT_PAID_VERSION_V1
EXPORT_PAID_VERSION_V1_1

Let's introduce a constant on String[] which will group the common profiles grouped:
EXPORT_PAID_INEHERITED_V1 = [V1, V1_1, V1_2]
EXPORT_PAID_INEHERITED_V1_1 = [V1_1, V1_2]
...

Reply:
ps: it's more than a suggestion: the presence of the field will create the column: you was creating CSV for older version with the new columns, simply they were null